docs/markdown/Build-options.md

```diff
@@ -107,6 +107,12 @@ $ meson configure "-Doption=['a,b', 'c,d']"
 The inner values must always be single quotes and the outer ones
 double quotes.
 
+To change values in subprojets prepend the name of the subproject and a colon:
```

Comment:
subprojets  -> subprojects

Reply:
Oups, thanks. Fixed column limit as well (not sure but seems to be 70?).